go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/nginxinc/nginx-go-crossplane
 
-go 1.19
+go 1.23.0
```

Comment:
this is a little surprising... I guess `golang.org/x/...` needs a newer go version now?